core/src/main/scala/tech/beshu/ror/accesscontrol/blocks/definitions/ldap/implementations/UnboundidLdapConnectionPoolProvider.scala

```diff
@@ -186,6 +176,31 @@ object UnboundidLdapConnectionPoolProvider extends Logging {
       .asyncBoundary
   }
 
+  private def testBinding(connectionConfig: LdapConnectionConfig) = {
+    val serverSet = createLdapServerSet(connectionConfig)
+    val bindReq = bindRequest(connectionConfig.bindRequestUser)
+    val maxRetries = if (connectionConfig.ignoreLdapConnectivityProblems) 0 else 5
+    val connectionTimeout: FiniteDuration = if (connectionConfig.ignoreLdapConnectivityProblems) {
```

Comment:
I don't understand it. Does it mean we should create a private, well-named method here too?

Reply:
I refactored it a little